scripts/modelbench/report.py

```diff
@@ -126,11 +126,12 @@ def percentage(part, whole, decimals=2):
 
 def update_passrate_csv(df, target_path, backend):
     new_df = df.copy()
+    precision = "amp" if args.precision == "amp_fp16" else args.precision

```

Comment:
Is this change a WA to get the right reference csv name?

Reply:
Yes. As the CSV name of the performance results is created by the `runner.py` script in the pytorch repo.